pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[project]
+name = "py-realstate"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.12"
+dependencies = [
+    "fastapi[standard]>=0.115.14",
+    "folium>=0.20.0",
+    "pytest>=8.4.1",
+    "requests[standard]>=2.32.4",
+    "ruff>=0.12.7",
+    "types-requests>=2.32.4.20250611",
```

Comment:
move dev dependencies to optional
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#dependencies-optional-dependencies

I.e. in dependencies you have only prod packages, and in optional your dev and other stuff